client/src/components/new-request/search.vue

```diff
@@ -1,32 +1,42 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal">
     <v-row cols="12">
-      <v-col cols="6" class="font-weight-bold h6">I need a name to:</v-col>
+      <v-col cols="6" class="font-weight-bold h6"><span>I need a name to:</span></v-col>
       <v-col cols="6">
         <span id="nr-required-activator"
               class="link-std d-flex justify-end"
               @click="activateNRRModal()">Check to see if you need to file a a Name Request</span>
       </v-col>
-      <v-col cols="5">
+      <v-col cols="5 mt-n1">
```

Comment:
Wait, you have CSS classnames in the `col` property?

Reply:
lol yea this works. So weird. Would you prefer in a `class=""`

Reply:
Yes please.